src/rpc/jsexprs.jl

```diff
@@ -97,6 +104,7 @@ function jsexpr(io, x::Expr)
   isexpr(x, :block) && return block_expr(io, rmlines(x).args)
   @match x begin
     d(xs__) => dict_expr(io, xs)
+    Dict(xs__) => dict_expr_06(io, x.args[2:end])
```

Comment:
If you only pass `x.args[2:end]`, is it even necessary to call `dict_expr_06` instead of `dict_expr`, which only differs by the offset index?